src/index.html

```diff
@@ -9,6 +9,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo__link">
+        <img
+          src="./images/logo.png"
+          alt="Moyo Header Logo"
+        >
+      </a>
```

Comment:
Here should be an empty space, this `a` tag is a sibling to `nav`, generally the rule is (apart from the exception with repeated tags) that the siblings should have an empty space between them:
```html
<div>
  <span></span>

  <p></p>

  <div>
    <ul>
      <li></li>
      <li></li>
    </ul>
  </div>
</div>
```